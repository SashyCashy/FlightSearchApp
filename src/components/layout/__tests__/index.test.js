import React from 'react';

import { shallow } from 'enzyme';
import Layout from '../index';
describe('<Layout /> ', () => {
  it('Check the layout structure ', () => {
    const wrapper = shallow(<Layout />);

    expect(wrapper).toMatchSnapshot();
  });
});
