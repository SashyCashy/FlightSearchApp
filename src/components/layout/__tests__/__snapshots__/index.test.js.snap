// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout />  Check the layout structure  1`] = `
<styled.div
  id="app__container"
>
  <LeftPane
    customerList={Array []}
    makeDataRequest={[Function]}
  />
  <RightPane
    address={Array []}
    customerDetails={Object {}}
    image=""
    name=""
  />
</styled.div>
`;
